common/util.go

```diff
@@ -530,6 +530,12 @@ func CreateHistoryStartWorkflowRequest(
 		firstDecisionTaskBackoffSeconds += delayStartSeconds
 	}
 
+	// Add a random jitter to start time, if requested.
+	jitterStartSeconds := startRequest.GetJitterStartSeconds()
+	if jitterStartSeconds > 0 {
+		firstDecisionTaskBackoffSeconds += rand.Int31n(jitterStartSeconds + 1)
```

Comment:
rand has a seed value somewhere?

Reply:
Good call - turns out we already seed rand in host/service.go for every service.

Reply:
perfect